tests/unit/discovery/py_info/test_py_info.py

```diff
@@ -56,7 +56,11 @@ def test_bad_exe_py_info_no_raise(tmp_path, caplog, capsys):
         list(
             "{}{}{}".format(impl, ".".join(str(i) for i in ver), arch)
             for impl, ver, arch in itertools.product(
-                ([CURRENT.implementation] + (["python"] if CURRENT.implementation == "CPython" else [])),
+                (
+                    [CURRENT.implementation]
+                    + (["python"] if CURRENT.implementation == "CPython" else [])
+                    + (["cpython"] if CURRENT.implementation == "CPython" else [])
```

Comment:
I think the correct logic here would be to add ``CURRENT.implementation.lower`` if that differs from ``CURRENT.implementation`` independent if it's CPython or not 👍